march_hardware/src/IMotionCube.cpp

```diff
@@ -32,6 +33,8 @@ void IMotionCube::mapMisoPDOs()
   PDOmap pdoMapMISO = PDOmap();
   pdoMapMISO.addObject(IMCObjectName::StatusWord);      // Compulsory!
   pdoMapMISO.addObject(IMCObjectName::ActualPosition);  // Compulsory!
+  pdoMapMISO.addObject(IMCObjectName::ActualTorque);
+  pdoMapMISO.addObject(IMCObjectName::TargetCurrent);
```

Comment:
Why is 1 called Torque and the other current?

Reply:
Remove TargetCurrent as miso? It's only necessary as mosi. ActualTorque is what you want to read as miso pdo

Reply:
Re @Ishadijcks : ActualTorque and TargetCurrent are the names as defined by Technosoft. Not much we can do about their inconsistency.